Query Builder/queries_run_export.php

```diff
@@ -77,23 +79,8 @@
         } else {
             $queryDetails = $result->fetch();
 
-            // Get bind values, if they exist
-            $data = [];
-            $bindValues = json_decode($queryDetails['bindValues'] ?? '', true);
-            if (!empty($bindValues) && is_array($bindValues)) {
-                foreach ($bindValues as $bindValue) {
-                    $fieldValue = $_GET[$bindValue['variable']] ?? '';
-                    if ($bindValue['type'] == 'date' && !empty($fieldValue)) {
-                        $fieldValue = Format::dateConvert($fieldValue);
-                    } elseif (is_array($fieldValue)) {
-                        $fieldValue = implode(',', $fieldValue);
-                    }
-                    $data[$bindValue['variable']] = $fieldValue;
-                }
```

Comment:
This section should likely stay in, as it's converting dates and arrays to be used in the query.

Reply:
Nevermind, I see now that you're passing the data array which is already converted, it was just being called bindValues.